sdk/storage/CONTRIBUTING.md

```diff
@@ -42,6 +42,8 @@ Example - `export TEST_MODE=record` or `set TEST_MODE=record`
   | ACCOUNT_SAS               | Can be obtained from the Azure portal. Go to the `Shared access signature` tab under the `Settings` section of your storage account in the portal. Generate SAS by enabling permissions, setting exiry dates, etc. You can also generate SAS from tools like Azure Storage Explorer. <br>A SAS starts with "?". And if you are using Windows CMD, you may need quotes to escape special characters like this - `set "ACCOUNT_SAS=<YOUR_SAS>"` | Required to run the browser tests                                                |
   | ACCOUNT_TOKEN             | Preferred way is to get a Token through Azure CLI.<br> - `az login`<br> - `az account get-access-token --resource "https://storage.azure.com"`<br>For more info - refer [Azure CLI - get-access-token](https://docs.microsoft.com/en-us/cli/azure/account?view=azure-cli-latest#az-account-get-access-token)                                                                                                                                  | Not required (Some of the tests will be skipped if this variable is not defined) |
   | STORAGE_CONNECTION_STRING | Can be obtained from the Azure portal. Go to the `Access Keys` tab under the `Settings` section of your storage account in the portal and note down the connection string.                                                                                                                                                                                                                                                                    | Required                                                                         |
+  | ENCRYPTION_SCOPE_1        | An encryption scope created under your storage account. Can be created using the SRP API.                                                                                                                                                                                                                                                                                                                                                     | Not required (Some of the tests will be skipped if this variable is not defined) |
```

Comment:
Minor: If we wish to use the env var approach. Could there be some introduction or link on what's encryption scope, and how encryption_scope_1/2 will be used?

Reply:
Better clarify what's the difference between encryption_scope_1 and encryption_scope_2?

Reply:
Encryption scope is a new feature, I am not sure its documentation is ready. Even the SRP API I used to create the encryption scope is not official yet.

Reply:
> Better clarify what's the difference between encryption_scope_1 and encryption_scope_2?

They are just two different encryption scope name. I will add comments to stress that they need to be different.